src/views/forms/CancelAllOrdersInMarketForm.tsx

```diff
@@ -130,21 +129,21 @@ export const CancelAllOrdersInMarketForm = ({
         value: (
           <DiffOutput
             type={OutputType.Fiat}
-            value={crossFreeCollateral?.current}
-            newValue={MustBigNumber(crossFreeCollateral?.current).plus(
-              MustBigNumber(thisPendingPosition?.freeCollateral?.current ?? 0)
+            value={crossFreeCollateral}
+            newValue={MustBigNumber(crossFreeCollateral).plus(
+              MustBigNumber(pendingPositionThisMarket?.equity ?? 0)
```

Comment:
i don't think an additional `MustBigNumber` is required in this case

`MustBigNumber(crossFreeCollateral).plus(pendingPositionThisMarket?.equity ?? 0)`